src/main/java/io/jenkins/docker/connector/DockerComputerSSHConnector.java

```diff
@@ -245,6 +245,7 @@ protected ComputerLauncher createLauncher(DockerAPI api, String workdir, Inspect
         // Wait until sshd has started
         // TODO we could (also) have a more generic mechanism relying on healthcheck (inspect State.Health.Status)
         final PortUtils.ConnectionCheck connectionCheck = PortUtils.connectionCheck( address );
+        PortUtils.ConnectionCheckSSH connectionCheckSSH = connectionCheck.useSSH();
```

Comment:
Code style: Declare this as `final`.
This could be final and the other could-be-final variables in this method are declared as `final` so this should be no different.

Reply:
sure